File: URI-Online-Judge-py/1160.py
import math

t=int(input())
a = input().split()
for i in range(0,t):
    PA = int(a[0])
    PB = int(a[1])
    G1 = float(a[2]) * 0.01
    G2 = float(a[3]) * 0.01
    j=0
    while PA<=PB:
        PA+=math.floor(PA*(G1))
        PB+=math.floor(PB*(G2))
        j+=1
        if (j > 100):
            break
    if(j<=100):
        print(j,"anos.")
    else:
        print("Mais de 1 seculo.")
    a = input().split()

File: URI-Online-Judge-c/1160.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

main(){
int t,i,pa,pb,ano;
double g1,g2;
scanf("%d",&t);
for(i=0;i<t;i++){
  scanf("%d %d %lf %lf",&pa,&pb,&g1,&g2);
  g1=g1*0.01;
  g2=g2*0.01;
  ano=0;
  while((pa<pb)||(pa==pb)){
    pa=pa+(pa*g1);
    pb=pb+(pb*g2);
    ano=ano+1;
    if (ano>100){
            break;
    }}
    if((ano<100)||(ano==100)){
        printf("%d anos.\n",ano);
    }
    else{
        printf("Mais de 1 seculo.\n");
    }
  }


return 0;
}
